stylus/components/tabs.styl

```diff
@@ -25,16 +26,27 @@ $tabs-base
         font-weight    bold
         opacity        .64
 
+        border-bottom $borderBottomWidth solid transparent
+
         &:hover
             text-decoration none
 
         +small-screen()
             &
```

Comment:
I know it's outside of the PR, but… isn't this useless?

Reply:
No, this is not useless, this way the text does not jump when the link is activated (otherwise, the border is "created" when the tab is activated and the text goes up 2px).

Reply:
I'll add a comment to clarify.

Reply:
See 1f0f143e

Reply:
Oh, I didn't mean the transparent border (though the comment is welcome), I meant the `&` on l.32?

Reply:
No, the & is compulsory, otherwise the :hover is seen as a child. See the result :

https://stylus-lang.com/try.html#?code=.toto%0A%09%26%3Ahover%0A%09%09background%3A%20red%0A%0A.toto%0A%09%3Ahover%0A%09%09background%3A%20blue
https://jsbin.com/maxenot/edit?html,css,js,output